custom_components/unifiprotect/binary_sensor.py

```diff
@@ -146,6 +164,38 @@ async def async_setup_entry(
 ) -> None:
     """Set up binary sensors for UniFi Protect integration."""
     entry_data: UnifiProtectEntryData = hass.data[DOMAIN][entry.entry_id]
+    entities = _async_device_entities(entry_data)
+    entities = _async_nvr_entities(entry_data)
+
+    async_add_entities(entities)
+
```

Comment:
Variable defined twice

Reply:
Haha. Oops. I guess I did not test the other binary sensors. They were probably all broken. I should make sure I do that.